torchaudio_contrib/functional.py

```diff
@@ -103,6 +103,30 @@ def stft(waveforms, fft_len, hop_len, window,
     return complex_specgrams
 
 
+def waveform_downmix(waveforms, ch_dim=1):
```

Comment:
I'd just call it `dim`. There is only one dimension to specify, and this is consistent with pytorch. Maybe someone may want to downmix across the batch dimension. Make sure it's included in the docstring, though. The docstring should also mention that it's downmixing by taking the mean.